ovh/resource_ovh_iploadbalancing_tcp_frontend.go

```diff
@@ -0,0 +1,224 @@
+package ovh
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+type IpLoadbalancingTcpFrontend struct {
+	FrontendId    int      `json:"frontendId,omitempty"`
+	Port          string   `json:"port"`
+	Zone          string   `json:"zone"`
+	AllowedSource []string `json:"allowedSource,omitempty"`
+	DedicatedIpFo []string `json:"dedicatedIpfo,omitempty"`
+	DefaultFarmId int      `json:"defaultFarmId,omitempty"`
+	DefaultSslId  int      `json:"defaultSslId,omitempty"`
+	Disabled      bool     `json:"disabled,omitempty"`
+	Ssl           bool     `json:"ssl,omitempty"`
+	DisplayName   string   `json:"displayName,omitempty"`
+}
```

Comment:
consider using pointers for values that you might need to update/set to their "empty" value. Otherwise you risk running into situation where you ie. created frontent with `disabled: true` now you want to enable it explicity setting `disabled: false` but... `false` is the emty/zero value for bool, so with `omitempty` this change will not get reflected on the API call.